From 1823de236f38fc7eb7caedec666c9dde956832cd Mon Sep 17 00:00:00 2001
From: Awais Belal <awais_belal@mentor.com>
Date: Wed, 31 May 2017 13:04:42 +0500
Subject: [PATCH] spirv: do not install conflicting headers

These headers are already provided by the
spirv-tools package so there's no need of
installing them here again.

Signed-off-by: Awais Belal <awais_belal@mentor.com>
---
 SPIRV/CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/SPIRV/CMakeLists.txt b/SPIRV/CMakeLists.txt
index aaf11780..d5c43723 100755
--- a/SPIRV/CMakeLists.txt
+++ b/SPIRV/CMakeLists.txt
@@ -12,8 +12,6 @@ set(SPVREMAP_SOURCES
 
 set(HEADERS
     bitutils.h
-    spirv.hpp
-    GLSL.std.450.h
     GLSL.ext.KHR.h
     GlslangToSpv.h
     hex_float.h
-- 
2.11.1

